homeassistant/components/recorder/core.py

```diff
@@ -779,6 +779,10 @@ def _run_event_loop(self) -> None:
         for task in startup_tasks:
             self._guarded_process_one_task_or_recover(task)
 
+        # Clear startup tasks since this thread runs forever
+        # and we don't want to hold them in memory
+        startup_tasks.clear()
```

Comment:
What about

```suggestion
        del startup_tasks
```

Reply:
I think that should work. Will merge it in and rerun the profile

Reply:
Works as expected 👍

Reply:
thanks